### PR TITLE
changed styles of home page to be responsive

### PR DESCRIPTION
**Pull request by o-t-w**
I noticed there was an issue for making the homepage responsive. 
https://github.com/whatwg/whatwg.org/issues/59

### CODE REVIEW

**Comment by annevk** [comment]
Could you maybe put this live somewhere in some form for ease of review? Unfortunately we don't have preview functionality for PRs in this repository.

I'm a little concerned with all the added inline SVG. Seems that would make this much harder to maintain.

**Comment by o-t-w** [comment]
Preview on Codepen 
https://s.codepen.io/cssgrid/debug/50554e6ae6f9a57e5d566b73edcbca37

**Comment by domenic** [comment]
Wow! This is a bold redesign. It's been a while with our current one though, and after my initial shock, and I think I like it!

I have a number of review comments, but I will save them until we get agreement that we want to go in this direction. (And maybe some of them I can fix myself.)

One thing I think we'd need though is help adapting the other pages which currently use the tabbed pages style, e.g. the FAQ. With this homepage redesign I'm still not sure how those would look.

**Comment by tabatkins** [comment]
I like this quite a lot!

My only nit is that I think the GitHub font is jarring; it looks out of place when surrounded by the other buttons just using normal text.  I think it should just be ordinary text as well.

Otherwise, A++++, this look dramatically better than the current homepage to me!

**Comment by annevk** [comment]
Thanks @o-t-w, I like this direction as well.

File: whatwg.org/index [hunk 1]
```diff
@@ -1,39 +1,93 @@
 <!DOCTYPE HTML>
 <html lang="en">
  <head>
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <meta http-equiv="X-UA-Compatible" content="ie=edge">
```
Comment (domenic) [inline whatwg.org/index]: Please don't add this; we don't need to help old-IE

File: whatwg.org/index [hunk 2]
```diff
@@ -1,39 +1,93 @@
...
-  <a href="https://wiki.whatwg.org/"><p><strong>Wiki</strong> <span>Poke at our wiki pages</span> <span>and see what you unearth</span></p></a>
-  <a href="https://twitter.com/WHATWG"><p><strong>Twitter</strong> <span>Keep track of spec changes</span> <span>by following our Twitter feed</span></p></a>
--->
-
-  <address>Queries can be directed either as a
-  <a href="https://github.com/whatwg/meta/issues/new">new meta issue</a> or as an email to
-  <a href="mailto:ian+whatwg@hixie.ch">Ian Hickson</a>, who is acting as a spokesperson for the
-  WHATWG.</address>
+ <body>
+<div class="wrapper">
```
Comment (domenic) [inline whatwg.org/index]: Stick with one-space indent please

File: whatwg.org/index [hunk 3]
```diff
@@ -1,39 +1,93 @@
...
+            <div class="flex">
+                    <img class="main-logo" src="https://resources.whatwg.org/logo.svg" height=100 width=100 alt>
+                    <div class="titles">
```
Comment (domenic) [inline whatwg.org/index]: Let's stick with hgroup/h1/h2

File: whatwg.org/index [hunk 4]
```diff
@@ -1,39 +1,93 @@
...
+                                <p>Maintaining and evolving HTML since 2004</p>     
+                    </div>
+                </div>
+                 
+             
+               <div class="span-all">
+                     <h2>The Standards</h2>
+                     <a class="standard" href="https://html.spec.whatwg.org/multipage/"><svg class="arrow" width="77" height="44" viewBox="0 0 77 44" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+                          <title>Group</title>
+                          <g id="C" transform="translate(202 164)">
```
Comment (domenic) [inline whatwg.org/index]: Can you extract these out into separate .svg files?

File: whatwg.org/index [hunk 5]
```diff
@@ -1,39 +1,93 @@
...
+                          </g>
+                          <defs>
+                          <path id="path0_fill" fill-rule="evenodd" d="M 16.4309 26.8506L 25.2302 35.7195C 27.1096 37.6138 27.1096 40.685 25.2302 42.5793C 23.3508 44.4736 20.3037 44.4736 18.4243 42.5793L 1.40955 25.4299C 0.46985 24.4827 -7.68049e-16 23.2414 -1.02865e-22 22C -5.00901e-16 20.7586 0.46985 19.5173 1.40955 18.5701L 18.4243 1.4207C 20.3037 -0.473568 23.3508 -0.473568 25.2302 1.4207C 27.1096 3.31498 27.1096 6.3862 25.2302 8.28047L 16.4309 17.1494L 72.1875 17.1494C 74.8454 17.1494 77 19.3211 77 22C 77 24.6789 74.8454 26.8506 72.1875 26.8506L 16.4309 26.8506Z"/>
+                          </defs>
+                          </svg>
+                          <h3>See the other standards developed at the WHATWG</h3></a>
+               </div>
+               <hr>
+               <div class="grid">
+                     <a href="/faq"><h2>FAQ</h2></a>
```
Comment (domenic) [inline whatwg.org/index]: These aren't h2s; they are just links. (They are not headings for a section or article.)

File: whatwg.org/index [hunk 6]
```diff
@@ -1,39 +1,93 @@
...
+                <h2>HTML checker</h2>
```
Comment (domenic) [inline whatwg.org/index]: Similarly these are not h2s. They can be `<strong>`, I guess, like the existing site

File: whatwg.org/index [hunk 7]
```diff
@@ -1,39 +1,93 @@
...
+                   <address>Queries can be directed either as a <a href="https://github.com/whatwg/meta/issues/new">new meta issue</a> or as an email to <a href="mailto:ian+whatwg@hixie.ch">Ian Hickson</a>, who is acting as a spokesperson for the WHATWG.</address>
+                   
+</div>    
   <script> var _gaq = [['_setAccount', 'UA-20955470-1'], ['_setDomainName', '.whatwg.org'], ['_trackPageview']]; </script>
   <script async src="https://ssl.google-analytics.com/ga.js"></script>
- </body>
-</html>
+
+</body>
+</html>
```
Comment (domenic) [inline whatwg.org/index]: Don't forget newline at end of file :)

**Comment by jgraham** [comment]
This looks great~ (well apart from WHATWG green, but that's hardly your fault :)

Some minor issues, that mostly turn out to be issues on the current site too:
* Is emailing Hixie really still a thing? Seems like we could remove that from the footer.
* "Maintaining and evolving HTML since 2004" seems to undersell what WHATWG does.
* Similarly, I feel like the emphasis on linking to HTML is misplaced. It's certainly the most complex standard in the WHATWG roster, but it would be nice to get more onto the front page e.g. something that looks more like
```
   Standards
   [HTML] [DOM] [URL] [Encoding] [Notifications] [Fullscreen] [Others…]
```

None of those things should block landing this, but it would be nice to fix them whilst there's some attention on the site.

**Comment by domenic** [comment]
Since the codepen link expired, here is a new preview that is mostly faithful to @o-t-w's work: https://jsbin.com/cijufadebe

I wish we had some ideas on how to adopt this style to the other pages besides the homepage. If we had that, I'd be happy to address all the above comments myself (including the ones I made) and get this merged.

**Pull request closed without merging**